tests/libTestUtils/TestUtils.cpp

```diff
@@ -59,6 +59,26 @@ uint8_t Dist1to99() { return RandomIntInRng<uint8_t>((uint8_t)1, (uint8_t)99); }
 
 PubKey GenerateRandomPubKey() { return PubKey(PrivKey()); }
 
+Address GetAddressFromPubKey(PubKey pubKey) {
```

Comment:
there already is Account::GetAddressFromPublicKey for such purpose, can directly call that

Reply:
As the classes under test are Account and Transaction, I think he made duplicate functions for `GetShardIndex` and `GetAddressFromPublicKey` to make sure they are computationally correct? @jendis can you confirm?

Reply:
It doesn't make much sense to me to make new function to examine the correctness of existing function with same utility. The idea for unit test is by checking the output whether is desired by the input however

Reply:
Well, Antonio told exactly what I meant. I think it make sense to keep it separate - it costs nothing but ensures the correctness of the computation in production code. I can imagine situation that can happen when inappropriate refactor in this production part is done.
On the other side, when any refactor is done intentionally in the end causes test failure, then it is absolutely right that you have to update even the unit test.
In this case I really copied the production code because I expected the code part computing the sha256 to be stable. Anyway I agree that it would make more sense to use directly the openssl lib and keep the constant ACC_ADDR_SIZE as a local constant. If this changes in constants.xml from any reason, then it is necessary to change it in test part, which I consider OK as I sad. If this changes "by accident", then test'll catch it.
So please let me know what do you think is the best way to go.

Reply:
I think in keeping with the current way of doing unit tests elsewhere, and to avoid the necessary changes in test part, we can just do as @kaikawaliu says and stick to calling the production code for now.